seaborn/tests/test_utils.py

```diff
@@ -281,3 +289,30 @@ def test_category_order():
 
         out = utils.categorical_order(x, ["b", "a"])
         nt.assert_equal(out, ["b", "a"])
+
+
+@network(url="https://github.com/mwaskom/seaborn-data")
```

Comment:
Does this decorator allow the test to get skipped when there's no network connection?